docs/src/pages/docs/functions/helpers.md

```diff
@@ -340,6 +340,52 @@ you provide an `undefined` values for the second, that `Pair` will not be
 represented in the resulting `Object`. Also, when if multiple keys share the
 same name, that last value will be moved over.
 
+
+#### getPropOr
+
+`crocks/helpers/getPropOr`
+
+```haskell
+getPropOr :: a -> (String | Integer) -> b -> a
+```
+
+If you want some safety around pulling a value out of an Object or Array with a
+single key or index, you can always reach for `getPropOr`. Well, as long as you
+are working with non-nested data that is. Just tell `getPropOr` either the key
+or index you are interested in, and you will get back a function that will take
+anything and return the wrapped value if the key/index is defined. If the
+key/index is not defined however, you will get back the provided default value.
+
```

Comment:
I know I didn't mention it on the others, but do we want to reference that this used to be called prop or is replacing prop?

Reply:
Oh. I like that, we will have to be removed when we release 1.0, but I am okay with that.

Will add. Should I update the other functions as well on this PR or would you think that would be too much noise?

Reply:
My preference is to be as specific as possible with PR's, that being said up to you, i think the change is small enough to slip in as part of this